### PR TITLE
adding pause and play button

### PR DESCRIPTION
**Pull request by Lausselloic**

### CODE REVIEW

File: scss/_carousel.scss [hunk 1]
```diff
@@ -104,7 +104,7 @@
   @include hover-focus {
     color: $carousel-control-color;
     text-decoration: none;
-    outline: 0;
+    outline: 2px dotted #000;
```
Comment (Lausselloic) [inline scss/_carousel.scss]: Divergence from bootstrap need to be commented
- add // boosted mod at the end of line
- don't use color code in scss file use var instead currently you could use $carousel-control-color who's coherent

File: scss/_o-carousel.scss [hunk 2]
```diff
@@ -28,7 +29,7 @@
   transition: width .5s ease;
 
   &:focus {
-    outline: 1px dotted #000;
+    outline: 2px dotted #000;
```
Comment (Lausselloic) [inline scss/_o-carousel.scss]: don't use color code in scss file use var instead currently you could use $carousel-control-color who's coherent or use btn styles ($input-focus-width and $input-focus-border-color)

File: site/docs/4.3/components/custom-carousel.md [hunk 3]
```diff
@@ -120,4 +127,23 @@ var mySwiper = new Swiper('.swiper-container', {
   }
 })
 ```
+Be aware that having a play/pause button implies having autoplay set to true in the configuration.<br>
```
Comment (Lausselloic) [inline site/docs/4.3/components/custom-carousel.md]: don't use <br> tag just let a blank line between sentences

File: site/_includes/scripts.html [hunk 4]
```diff
@@ -80,7 +83,50 @@
 
   $('#collapsing-navbarHead').megamenu();
 
+  // Swiper pause and play
+  $(document).ready(function () {
+  var MySwiper = document.querySelector('.swiper-container').swiper;
```
Comment (Lausselloic) [inline site/_includes/scripts.html]: and what's append on pages where there's no swiper container?
=> a JS error
- check before if `.swiper-container` element exist onto the page please

File: site/docs/4.3/components/custom-carousel.md [hunk 5]
```diff
@@ -49,10 +49,14 @@ This carousel is based on the [swiper](http://idangero.us/swiper/) plugin. It pr
             </div>
         </div>
     </div>
+    <div class="swiperButton">
+        <button id="pauseSwiperButton" type="button" class="btn btn-default btn-xs swiper-button" aria-label="Pause Carousel" title="Pause Carousel">
```
Comment (Lausselloic) [inline site/docs/4.3/components/custom-carousel.md]: focus is not visble on that element when browsing with keyboard

File: site/docs/4.3/components/carousel.md [hunk 6]
```diff
@@ -62,18 +62,44 @@ Adding in the previous and next controls:
...
     <span class="carousel-control-prev-icon" aria-hidden="true"></span>
     <span class="sr-only">Previous</span>
   </a>
-  <a class="carousel-control-next" href="#carouselExampleControls" role="button" data-slide="next">
+  <a class="carousel-control-next" href="#carouselExampleControls" role="button" data-slide="next" title="Next">
     <span class="carousel-control-next-icon" aria-hidden="true"></span>
     <span class="sr-only">Next</span>
   </a>
+  <div class="carouselButton">
+    <button id="pauseCarouselButton" type="button" class="btn btn-default btn-xs carousel-button" aria-label="Pause Carousel" title="Pause Carousel">
```
Comment (Lausselloic) [inline site/docs/4.3/components/carousel.md]: focus is not visible on that element, carousel-button class is defined in o-carousel which is dedicated to custom carousel.

File: site/_includes/scripts.html (hunk 4 above)
Comment (liyokuna) [reply site/_includes/scripts.html]: oh waoh! I forget that... It was suposed to be inside the click event function.

**Comment by Aniort** [comment]
everything is OK for me 
Can publish !

**Pull request merged**